modular_skyrat/modules/primitive_catgirls/code/map_items.dm

```diff
@@ -23,12 +8,92 @@
 	grill = TRUE
 	add_overlay("bonfire_grill")
 
-// Dirt but icebox
+// Dirt but icebox and also farmable
 
 /turf/open/misc/dirt/icemoon
 	baseturfs = /turf/open/openspace/icemoon
 	initial_gas_mix = "ICEMOON_ATMOS"
 
+/turf/open/misc/dirt/icemoon/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/simple_farm, set_plant = TRUE)
+
+// Water that can be fished out of
+
+/turf/open/water/icebox_fishable
+	desc = "Water kept warm through some unknown heat source, possibly a geothermal heat source far underground. \
+		Whatever it is, it feels pretty damn nice to swim in given the rest of the environment around here, and you \
+		can even catch a glimpse of the odd fish darting through the water."
+	baseturfs = /turf/open/openspace/icemoon
+	initial_gas_mix = "ICEMOON_ATMOS"
+	/// Holder for the steam particles that show up sometimes
+	var/obj/effect/abstract/particle_holder/particle_effect
+
+/turf/open/water/icebox_fishable/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/lazy_fishing_spot, /datum/fish_source/icecat_hot_spring)
+	if(prob(60))
+		particle_effect = new(src, /particles/hotspring_steam)
+
+/turf/open/water/icebox_fishable/Destroy()
+	QDEL_NULL(particle_effect)
+	return ..()
+
+/turf/open/water/icebox_fishable/Entered(atom/movable/arrived)
+	..()
+	wash_atom(arrived)
+	wash_atom(loc)
+
+// Cleans the given atom of whatever dirties it
+/turf/open/water/icebox_fishable/proc/wash_atom(atom/nasty)
```

Comment:
Documentation is with three `/` (`///`), not just two. You've done this in a few times in this PR, whoops!